jcenter_test.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"os"
+	"path"
+	"testing"
+
+	"github.com/jfrog/jfrog-cli-go/artifactory/utils"
+	"github.com/jfrog/jfrog-cli-go/utils/tests"
+)
+
+func TestGetJcenterRemoteDetails(t *testing.T) {
```

Comment:
I'm not sure we need to split tests that are related to jcenter to a separate file.

Reply:
I will place jcenter's tests in artifactory's test.